### PR TITLE
Bluetooth: Fix unused HCI command response handling

### PR DESCRIPTION
**Pull request by LingaoM**
see commit

### CODE REVIEW

File: tests/bluetooth/hci_codecs_info/src/main.c
Comment (jhedberg) [inline tests/bluetooth/hci_codecs_info/src/main.c:1]: This change is in a commit called `drivers: bluetooth: Remove unused rsp params`. Please move it into a separate commit (e.g. `tests: bluetooth: ...`).

**Review by jhedberg — CHANGES_REQUESTED** [review]

**Review by jhedberg — APPROVED** [review]

**Review by Thalley — APPROVED** [review]
